bin/beeline

```diff
@@ -23,8 +23,29 @@
 # Enter posix mode for bash
 set -o posix
 
+realpath () {
```

Comment:
I will continue to think a way to remove these annoying duplication, also do more corner tests about this function.